galette/templates/default/elements/footer.html.twig

```diff
@@ -38,8 +39,15 @@
             </div>
             <div class="row">
                 <div class="ui horizontal bulleted link list">
-                    <a id="copyright" href="https://galette.eu/" class="item">
+                    <a id="copyright" href="https://galette.eu/" class="item{% if new_release.new %} tooltip" data-title="{{ _T("A new Galette release is available.") }}" data-content="{{ _T("You currently use Galette %1$s, and %2$s is available.")|replace({"%1$s": constant('GALETTE_DISPLAY_VERSION'), "%2$s": new_release.version}) }}"{% else %}"{% endif %}>
+{% if new_release.new and login.isLogged() and (login.isAdmin() or login.isStaff()) %}
+                        <div class="ui transition looping pulsating blue circular horizontal label">
```

Comment:
```suggestion
                        <div class="ui transition looping pulsating blue circular horizontal icon label">
```

Without the "icon" class, I have a display glitch:
![image](https://github.com/galette/galette/assets/224733/b37ff519-7d74-40ac-b41b-f3cc9b7a9dc5)